### PR TITLE
Rename multiple attachments

### PR DESCRIPTION
**Pull request by RubenGarcia**
### First time contributor checklist:

- [ X] I have read the [README](https://github.com/oxen-io/session-desktop/blob/master/README.md) and [Contributor Guidelines](https://github.com/oxen-io/session-desktop/blob/master/CONTRIBUTING.md)

### Contributor checklist:

- [ X] My commits are in nice logical chunks with [good commit messages](http://chris.beams.io/posts/git-commit/)
- [ X] My changes are [rebased](https://blog.axosoft.com/golden-rule-of-rebasing-in-git/) on the latest [`clearnet`](https://github.com/oxen-io/session-desktop/tree/clearnet) branch
- [ X] A `yarn ready` run passes successfully ([more about tests here](https://github.com/oxen-io/session-desktop/blob/master/CONTRIBUTING.md#tests))
- [ X] My changes are ready to be shipped to users

### Description

Fixes https://github.com/oxen-io/session-desktop/issues/2743
by always passing the attachment number to the function which creates the filename.
I tested it in Ubuntu 20.04.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,6 +82,7 @@
     "bunyan": "^1.8.15",
     "bytebuffer": "^5.0.1",
     "classnames": "2.2.5",
+    "cmake-js": "^7.2.1",
```
[the remark this answers is not in the document]
Comment (Bilb) [reply package.json:85]: we can keep the contributing.md changes, but CI doesn't need this

File: ts/util/attachmentsUtil.ts [hunk 2]
```diff
@@ -387,18 +387,21 @@ export const saveAttachmentToDisk = async ({
   messageTimestamp,
   messageSender,
   conversationId,
+  index,
 }: {
   attachment: AttachmentType;
   messageTimestamp: number;
   messageSender: string;
   conversationId: string;
+  index: number;
```
Comment (Bilb) [inline ts/util/attachmentsUtil.ts:396]: can you make the `  index?: number;` not optional in the function `export const save = ({` in the file `ts/types/Attachment.ts` too please?

**Review by Bilb — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (RubenGarcia) [reply package.json:85]: > Hey, this should not be needed, libsession util-nodejs is the only package needed this, and cmake-js is already a dep of libsession-util-nodejs

It's needed in linux, but I'm happy to remove it and have a different patch address this.

File: ts/util/attachmentsUtil.ts (hunk 2 above)
Comment (RubenGarcia) [reply ts/util/attachmentsUtil.ts:396]: I can, but this function is called at other places which require the default value to be calculated.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (RubenGarcia) [reply package.json:85]: done

**Pull request merged**